types/selenium-webdriver/ie.d.ts

```diff
@@ -7,11 +14,14 @@ export class Driver extends webdriver.WebDriver {
   /**
    * Creates a new session for Microsoft's Internet Explorer.
    *
-   * @param {(capabilities.Capabilities|Options)=} opt_config The configuration
-   *     options.
+   * @param {(Capabilities|Options)=} options The configuration options.
+   * @param {(remote.DriverService)=} opt_service The `DriverService` to use
+   *   to start the IEDriverServer in a child process, optionally.
    * @return {!Driver} A new driver instance.
    */
-  static createSession(opt_config?: webdriver.Capabilities|Options): Driver;
+  static createSession(
+    options: webdriver.Capabilities|Options,
```

Comment:
Is `options` no longer optional? The JSDoc comment above and the [implementation](https://github.com/SeleniumHQ/selenium/blob/5fe20e7658148a4fff90f612556853746bca1cad/javascript/node/selenium-webdriver/ie.js#L392-L393) seem to indicate it is still optional.

Reply:
Good catch, thanks! I assumed it was non-optional based on the implementation using the name `options` instead of the `opt_config` the other browsers' equivalents use, but I agree that it should be optional based on the JSDoc and implementation. Updated.